### PR TITLE
Add wish-list layout

### PR DESCRIPTION
**Pull request by manojkumar8400**
In wish list we keep our favourite things.

### CODE REVIEW

**Review by sakshamAK — APPROVED** [review]
image

You should change this cart's color. Rest is all good

**Comment by manojkumar8400** [comment]
> image
> 
> You should change this cart's color. Rest is all good

Yes will do that.

**Comment by pritam-kr** [comment]
Good to go

**Review by pritam-kr — APPROVED** [review]

File: common.css [hunk 1]
```diff
@@ -6,6 +6,8 @@
 :root{
     --primary-color:rgb(192, 156, 103);
     --secondary-color:rgb(168, 144, 128);
+    --footer-container:rgb(205, 206, 189);
+    --detail-cart:rgb(164, 155, 170);
     --white:#fff;
     --gray-200:lightgray;
     --gray-500:gray;
```
Comment (tanveertkd) [inline common.css:13]: Try keeping the colors in one format, like hex or rgba. You can find the converted values in vscode itself. This just helps with consistency.

**Review by tanveertkd — APPROVED** [review]
I noticed the cart and homepage are not responsive but I'm assuming you're going to add it later on.

File: common.css (hunk 1 above)
Comment (manojkumar8400) [reply common.css:13]: ok I will do it.

**Comment by manojkumar8400** [comment]
> I noticed the cart and homepage are not responsive but I'm assuming you're going to add it later on.

I will work on the responsiveness after today's class.

**Pull request merged**